src/stores/PostStore.ts

```diff
@@ -148,6 +146,17 @@ farcasterContract.on(
     PostIdsStatuses.statuses[id.toNumber()] = Promise.resolve(
       PostStatus.pending
     )
+
+    /* Check if sender is the `replayAll` address */
```

Comment:
I think we don't need this comment, the 
`    if (sender !== replyToAll) return`
Is enought self-explanatory